app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+  before_save :comment_counter
+
+  def comment_counter
+    post.update(comments_counter: post.comments.all.length)
+  end
```

Comment:
- [ ] **[OPTIONAL]** I would recommend you make this method private and we can save some code here.

  ```
  private

  def comment_counter
    post.increment!(:comments_counter)
  end
  ```